app/frontend/src/features/userPage/ResponsiveAvatar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { User } from "../../pb/api_pb";
+import { makeStyles } from "@material-ui/core";
+import Avatar from "../../components/Avatar";
+
+const useStyles = makeStyles({
+  root: {
+    position: "relative",
+    width: "100%",
+    paddingTop: "100%",
+  },
+  avatar: {
+    position: "absolute",
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    width: "100%",
+    height: "100%",
+  },
+});
+
+export default function ReponsiveAvatar({ user }: { user: User.AsObject }) {
```

Comment:
```suggestion
export default function ResponsiveAvatar({ user }: { user: User.AsObject }) {
```
But do you still need to use this Avatar now I made changes to the Avatar in Components?